### PR TITLE
Refactoring create product with several websites and check url rewrites test

### PR DESCRIPTION
**Pull request by kate-kyzyma**
### Description (*)
 AdminCreateProductWithSeveralWebsitesAndCheckURLRewritesTest is refactored according to the best practices followed by MFTF.

### Fixed Issues (if relevant)
N/A

### Manual testing scenarios (*)
1. Run the StorefrontLoginWithIncorrectCredentialsTest in the local environment and verified the test runs successfully.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by dmytro-ch** [comment]
@magento run all tests

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSubmitCategoriesPopupActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSubmitCategoriesPopupActionGroup">
+        <annotations>
+            <description>Clicks on Done on the search categories popup.</description>
```
Comment (dmytro-ch) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSubmitCategoriesPopupActionGroup.xml]:
```suggestion
            <description>Clicks the "Done" button on the Search Categories popup.</description>
```

File: app/code/Magento/UrlRewrite/Test/Mftf/ActionGroup/AssertAdminStoreValueIsSetForUrlRewriteActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminStoreValueIsSetForUrlRewriteActionGroup">
+        <annotations>
+            <description>Validates that the proper Store Value is used for URL Rewrite.</description>
```
Comment (dmytro-ch) [inline app/code/Magento/UrlRewrite/Test/Mftf/ActionGroup/AssertAdminStoreValueIsSetForUrlRewriteActionGroup.xml]:
```suggestion
            <description>Verifies that the proper Store Value is used for URL Rewrite.</description>
```

File: app/code/Magento/UrlRewrite/Test/Mftf/ActionGroup/AssertAdminStoreValueIsSetForUrlRewriteActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        </annotations>
+        <arguments>
+            <argument name="storeValue" type="string"/>
+        </arguments>
+
+        <see selector="{{AdminUrlRewriteIndexSection.gridCellByColumnRowNumber('1', 'Store View')}}" userInput="{{storeValue}}" stepKey="seeStoreValueForCategoryId"/>
```
Comment (dmytro-ch) [inline app/code/Magento/UrlRewrite/Test/Mftf/ActionGroup/AssertAdminStoreValueIsSetForUrlRewriteActionGroup.xml]:
```suggestion
        <see selector="{{AdminUrlRewriteIndexSection.gridCellByColumnRowNumber('1', 'Store View')}}"
                  userInput="{{storeValue}}" stepKey="seeStoreValueForCategoryId"/>
```

**Review by dmytro-ch — CHANGES_REQUESTED** [review]
Hello @kate-kyzyma, well done! The overall result looks good to me.
Could you please apply minor improvements?
Thank you!

**Comment by dmytro-ch** [comment]
@magento run all tests

**Comment by dmytro-ch** [comment]
@magento run all tests

**Review by dmytro-ch — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch, thank you for the review.
[ENGCOM-7606](https://jira.corp.magento.com/browse/ENGCOM-7606) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**